### ISSUE TITLE
[webview_flutter] Support setting useWideViewPort

### ISSUE THREAD

**Issue by Aulig** [flutter/flutter#93125]
Some websites (e.g. those built with the wix website builder) don't really work without useWideViewPort on Android. If you don't use wide view port you will have a weird white bar on one or both sides.

As an example, here are 2 screenshots from wix.com:

How it should look (screenshot taken from Firefox):
![signal-2021-11-05-145810_001](https://user-images.githubusercontent.com/32148066/140522244-7a7538ea-25a2-45ff-a1c7-49914d76e02f.png)

How it looks in webview_flutter without wideviewport:
![grafik](https://user-images.githubusercontent.com/32148066/140522314-2d062e8f-e9d2-47e0-baa8-6307d462e84f.png)

Therefore it would be nice if the webview supported setting useWideViewPort.

**Comment by stuartmorgan** [comment]
The fix for #70731 added [unconditionally enabling useWideViewPort](https://github.com/flutter/plugins/blob/bb62aaa4f384772152572bcbe710571dc3b1cd35/packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewBuilder.java#L165). Can you explain in more detail what you are requesting here that is different than that?

**Comment by stuartmorgan** [comment]
> How it looks in webview_flutter without wideviewport

What version of `webview_flutter` and `webview_flutter_android` is that screenshot taken with?

**Comment by Aulig** [comment]
Never mind then - my mistake. I checked https://github.com/flutter/plugins/pull/3325/files and didn't see any change that showed adding useWideViewPort.

It was also crossed out in the PR, so I thought it was undone:
![grafik](https://user-images.githubusercontent.com/32148066/140530410-df9f9f2d-99ef-480d-8639-4c0fc649f442.png)

Sorry for wasting your time. I've confirmed that wix websites work with webview_flutter 2.3.0
